experimental/sdg-tracker/package.json

```diff
@@ -26,14 +27,19 @@
     "styled-components": "^6.0.6"
   },
   "devDependencies": {
-    "@types/react": "^17.0.2",
     "@types/react-dom": "^17.0.2",
+    "@types/react": "^17.0.2",
     "@typescript-eslint/eslint-plugin": "^6.0.0",
     "@typescript-eslint/parser": "^6.0.0",
     "@vitejs/plugin-react-swc": "^3.3.2",
-    "eslint": "^8.45.0",
+    "eslint-config-prettier": "^6.11.0",
```

Comment:
Can you remove the newly added dependencies here that arent actually used by the linter for now? It might be all of the new dependencies, but at least prettier &  import-sort

Reply:
Removed all newly added dev dependencies.